packages/client/src/components/app/forms/MultiFieldSelect.svelte

```diff
@@ -43,7 +56,7 @@
 >
   {#if fieldState}
     <CoreMultiselect
-      value={fieldState.value || []}
+      value={expandedValue}
```

Comment:
Like mentioned above, if we leave `value` as it was and instead pass in our `expandedDefaultValue` to the `defaultValue` prop of the `Field` component instead, then we should be good to go 👍 
```js
defaultValue={expandedDefaultValue}
```

Reply:
as mentioned above, I have expanded both value and defaultValue now.